serverless/functions/serverless-functions-developer-guide.adoc

```diff
@@ -0,0 +1,22 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-developer-guide"]
+= Developer guide
```

Comment:
This title seems a little generic. Do you think this could be updated to match our gerund style titles? Like `Developing Functions`, or something similar?

Reply:
I'll change this back to Getting Started w/ Functions I guess since there's another upcoming topic that would suit Developing Functions better 🙂